### DISCUSSION TITLE
[Feature] sort assets by albums on device when adding to immich album

### DISCUSSION THREAD

**Discussion by Behem07h** [immich-app/immich#6614] [Feature Request]
### The feature

Currently when adding assets to an immich album, assets are only sorted by date. This makes it tedious to select assets that span a long period of time

It would be much easier to choose images if you could sort them by album on device instead of just date.

### Platform

- [ ] Server
- [x] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678
